socorro/processor/rules/breakpad.py

```diff
@@ -303,35 +278,34 @@ def __repr__(self):
 
     def predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
         if (
-            processed_crash.product != "Firefox"
-            or not processed_crash.os_name.startswith("Windows")
-            or processed_crash.cpu_arch != "x86"
+            processed_crash.get("product", "") != "Firefox"
+            or not processed_crash.get("os_name", "").startswith("Windows")
+            or processed_crash.get("cpu_arch", "") != "x86"
         ):
             # we don't want any of these
             return False
 
-        frames = (
-            processed_crash.get("json_dump", {})
-            .get("crashing_thread", {})
-            .get("frames", [])
+        frames = glom.glom(
+            processed_crash, "json_dump.crashing_thread.frames", default=[]
         )
         if frames and frames[0].get("module", False):
             # there is a module at the top of the stack, we don't want this
             return False
 
+        signature = processed_crash.get("signature", "")
```

Comment:
This rule needs to run after mdsw and after signature generation. That should be denoted in the class docstring.